builtin/logical/aws/rotation_test.go

```diff
@@ -0,0 +1,193 @@
+package aws
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/iam"
+	"github.com/hashicorp/go-secure-stdlib/awsutil"
+	"github.com/hashicorp/vault/sdk/logical"
+	"github.com/hashicorp/vault/sdk/queue"
+)
+
+// TestRotation verifies that the rotation code and priority queue correctly selects and rotates credentials
+// for static secrets.
+func TestRotation(t *testing.T) {
+	bgCTX := context.Background()
```

Comment:
nit: rename as `ctx`

Reply:
I actually kind of prefer bgCTX here since it was created specifically to avoid calling `context.Background()` each time. If we want a different context in the tests, we're probably doing something different and would want a separate non-background context.